_posts/2022-03-16-cockpit-265.md

```diff
@@ -0,0 +1,76 @@
+---
+title: Cockpit 265
+author: skobyda
+date: '2022-03-16'
+tags: cockpit, machines
+slug: cockpit-265
+category: release
+summary: 'Animated lists, X-Forwarded-For support, crypto policies, disk serial number, metrics showing busiest CPU core, manifest overrides location'
+---
+
+Cockpit is the [modern Linux admin interface](https://cockpit-project.org/).
+We release regularly.
+
+Here are the release notes from Cockpit 265 and cockpit-machines 264:
+
+
+## Animate new rows in lists
+
+When new items appear on a page, they are highlighted for a short while to spot them more easily.
+
+![Peek 2022-03-15 07-47]({{ site.baseurl }}/images/265-animate-new-rows-in-lists.gif)
+
+## Support for `X-Forwarded-For`
+
+When run in a configuration involving a reverse proxy server, Cockpit can now report the client IP address as provided to it by the proxy server, rather than using the IP of the proxy server itself.  This impacts what appears in system login logs, for example, and may have an impact on authentication, depending on which pam modules are in use.
+
+Support for this feature must be explicitly enabled via the new `ForwardedForHeader` option in the [configuration file](https://cockpit-project.org/guide/latest/cockpit.conf.5.html).
+
+## Manifest overrides in /etc and ~/.config directories
+
+[Manifest override files](https://cockpit-project.org/guide/latest/packages.html#package-replace) are a way to customize or hide Cockpit's page menu entries. Until now these needed to be put next to the `manifest.json` which they modify, for example `/usr/share/cockpit/systemd/override.json`. However, `/usr/share/` is package manager territory,  and admins should not (and in some cases even cannot) write files there. With this Cockpit version, these overrides can now be placed into `/etc/cockpit/systemd.override.json` for global customizations, or `~/.config/cockpit/systemd.override.json` for user specific ones.
+
+## Crypto policies support
+
+Cockpit now has basic support for displaying and changing a subset of crypto policies (default, future, legacy). The current configured crypto policy is shown in the "Configuration" card.
+
+![158211328-fd8936be-5127-4e30-9c4b-5ba251c7d60a]({{ site.baseurl }}/images/265-crypto-policies-support.png)
```

Comment:
This image is blurry, and it couldn't hurt to equalize the frame width a bit.

Also, please let's not use these UUIDs as alt names, but something readable like "crypto policies in Configuration card"

Reply:
Please still fix these horrible UUIDs.